elasticsearch/examples/networkpolicy/test/goss.yaml

```diff
@@ -0,0 +1,39 @@
+kernel-param:
```

Comment:
If there is no specific test relative to network policies, the `test` target and `goss.yaml` isn't necessary for this example.

Reply:
Should it be something like
```
test: install
	helm test $(RELEASE)
```

then?